base/util.jl

```diff
@@ -320,13 +320,21 @@ end
 function with_output_color(f::Function, color::Union{Int, Symbol}, io::IO, args...; bold::Bool = false)
     buf = IOBuffer()
     iscolor = get(io, :color, false)
-    iscolor && bold && print(buf, text_colors[:bold])
-    iscolor && print(buf, get(text_colors, color, color_normal))
     try f(IOContext(buf, io), args...)
     finally
-        iscolor && color != :nothing && print(buf, get(disable_text_style, color, text_colors[:default]))
-        iscolor && (bold || color == :bold) && print(buf, disable_text_style[:bold])
-        print(io, String(take!(buf)))
+        lines = split(String(take!(buf)), '\n')
+        first = true
+        iob = IOBuffer()
+        for line in lines
+            first || print(iob, '\n')
+            first = false
```

Comment:
Perhaps check if line is empty here and `continue` do not write sequences on empty lines.

Reply:
Done